### ISSUE TITLE
python 2.7.3-7 incorrectly softlinks

### ISSUE THREAD

**Issue by yannpaul** [ContinuumIO/anaconda-issues#308]
So this is probably not a critical issue, but I can't create envs with python-2.7.3-7 using soft links but I can with hard links.

## Background

My home directory is on one file server and my workspace is on another. CONDA_ENVS_PATH points to a dir in my home and that's where all downloaded packages are stored. When I create an env in my workspace softlinks are used by default. This doesn't work for python-2.7.3-7. 

Separately I tried to make my own build of Python-2.7.9, one that uses a more up-to-date readline to deal with #152. That package had the same issue until I added a 'no_link' field in the recipe. 

## Proposed Solution

I think the python-2.7.3-7 conda package is missing a 'no-link' value in the build section of the package's recipe. If I look at the pythons I have in my '.pkgs' dir, python-2.7.3-7 is the only one missing a 'no_softlink' file in the package's info/ dir. 

I ls'd for 'files', which all packages should have, and 'no_softlink' in each python's info/ dir: 

    python-2.6.9-1/info/files
    python-2.6.9-1/info/no_softlink
    python-2.7.3-7/info/files
    python-2.7.7-0/info/files
    python-2.7.7-0/info/no_softlink
    python-2.7.8-0/info/files
    python-2.7.8-0/info/no_softlink
    python-2.7.8-1/info/files
    python-2.7.8-1/info/no_softlink
    python-2.7.9-1/info/files
    python-2.7.9-1/info/no_softlink
    python-2.7.9-2/info/files
    python-2.7.9-2/info/no_softlink

So, in short, something like this needs to be added to the Python-2.7.3-7 recipe:

    build:
    #  other stuff
        no_link:
            - bin/python
            - bin/python-config
            - bin/python2
            - bin/python2.7

**Comment by yannpaul** [comment]
FYI adding the 'no_link' section to my own build of python did not work. I'm not sure what's missing...
